src/stackedhist.jl

```diff
@@ -0,0 +1,32 @@
+@userplot StackedHist
+
+@recipe function f(h::StackedHist)
+    arrays = reverse(h.args[1])
+    elt = eltype(arrays)
+    if elt <: Histogram 
+        bins = arrays[1].edges[1] # edges is a tuple
+        weights_ary = hcat([x.weights for x in arrays]...) #reverse so it stacks from top to bottom
+    elseif elt <: Array{<:Real} # if raw datas
+        # if no bins given, guess by fitting first data array
+        bins = pop!(plotattributes, :bins, fit(Histogram, arrays[1]).edges[1])
+        hist_wgts = pop!(plotattributes, :weights, [nothing]) |> reverse
+        weights_ary = (
+            hist_wgts == [nothing] ?
+                [fit(Histogram, x, bins).weights for x in arrays] :
+                [fit(Histogram, x, weights(y), bins).weights for (x, y) in zip(arrays, hist_wgts)]
+            )
+        weights_ary = hcat(weights_ary...)
+            
+    else
+        error("args[1] should be collection of data arrays or of histograms")
+    end
+    
+    stacked_data = abs.(reverse(cumsum(weights_ary, dims = 2), dims=2))
+    ymin = 10^floor(log10(minimum(stacked_data)))
```

Comment:
sorry not quite getting the idea with the exponents of 10?

Reply:
`ylim` is to avoid it goes wrong in logscale:
`ArgumentError: At least one finite value must be provided to formatter.`